pkg/crypto/hash/hash_test.go

```diff
@@ -47,3 +47,9 @@ func TestHash160(t *testing.T) {
 	actual := hex.EncodeToString(data.Bytes())
 	assert.Equal(t, expected, actual)
 }
+
+func TestChecksum(t *testing.T) {
+	a := Checksum([]byte{1})
+	b := Checksum([]byte{2})
+	assert.NotEqual(t, a, b)
```

Comment:
It's valid, but kinda weak test, how about adding some known test vectors? We have a very particular function that generates these checksums and it should always produce the same good known results.